test/danetest.c

```diff
@@ -22,12 +22,16 @@
 #ifndef OPENSSL_NO_ENGINE
 #include <openssl/engine.h>
 #endif
+#include "testutil.h"
+#include "test_main_custom.h"
 
-#include "../e_os.h"
+#include "e_os.h"
```

Comment:
well, making it more like everything else then.
i'm all for consistency, but a separate PR.